animations/tween-animations.md

```diff
@@ -0,0 +1,727 @@
+---
+layout: page
+title: Tween Animations in Flutter
+permalink: /animations/tween-animations.html
+---
+
+* TOC Placeholder
+{:toc}
+
+The animation system in Flutter is based on typed
+[`Animation`](https://docs.flutter.io/flutter/animation/Animation-class.html)
+objects. Widgets can either incorporate these animations in their build
+functions directly by reading their current value and listening to their
+state changes or they can use the animations as the basis of more elaborate
+animations that they pass along to other widgets.
```

Comment:
When we say "animation" here, it'd be helpful to distinguish between the general concept of animation and Dart objects with the Animation type.